tests/integration/components/gh-image-uploader-with-preview-test.js

```diff
@@ -1,46 +1,41 @@
 /* jshint expr:true */
 import {expect} from 'chai';
-import {
-    describeComponent,
-    it
-} from 'ember-mocha';
+import {describe, it} from 'mocha';
+import {setupComponentTest} from 'ember-mocha';
 import hbs from 'htmlbars-inline-precompile';
 import run from 'ember-runloop';
 import sinon from 'sinon';
 
-describeComponent(
-    'gh-image-uploader-with-preview',
-    'Integration: Component: gh-image-uploader-with-preview',
-    {
+describe('Integration: Component: gh-image-uploader-with-preview', function() {
+    setupComponentTest('gh-image-uploader-with-preview', {
         integration: true
-    },
-    function() {
-        it('renders image if provided', function() {
-            this.set('image', 'http://example.com/test.png');
+    });
 
-            this.render(hbs`{{gh-image-uploader-with-preview image=image}}`);
+    it('renders image if provided', function() {
+        this.set('image', 'http://example.com/test.png');
 
-            expect(this.$('.gh-image-uploader.-with-image').length).to.equal(1);
-            expect(this.$('img').attr('src')).to.equal('http://example.com/test.png');
-        });
+        this.render(hbs`{{gh-image-uploader-with-preview image=image}}`);
 
-        it('renders upload form when no image provided', function () {
-            this.render(hbs`{{gh-image-uploader-with-preview image=image}}`);
+        expect(this.$('.gh-image-uploader.-with-image').length).to.equal(1);
+        expect(this.$('img').attr('src')).to.equal('http://example.com/test.png');
+    });
 
-            expect(this.$('input[type="file"]').length).to.equal(1);
-        });
+    it('renders upload form when no image provided', function () {
+        this.render(hbs`{{gh-image-uploader-with-preview image=image}}`);
 
-        it('triggers remove action when delete icon is clicked', function () {
-            let remove = sinon.spy();
-            this.set('remove', remove);
-            this.set('image', 'http://example.com/test.png');
+        expect(this.$('input[type="file"]').length).to.equal(1);
```

Comment:
@Turbo87 This is the failing test right here, and after inserting a console.log statement I found the error.

It appears the component is retaining the value from this line: `this.set('image', 'http://example.com/test.png');` in the `it` above.

Is it an incorrect understanding that each `it` is supposed to be run with a clean slate? i.e. `this.get('image')` should return undefined for the second it because it hasn't been explicitly set in the current test.

If I'm incorrect in that understanding, adding a `this.set('image', null);` above this line should fix the failing test.